### PR TITLE
(role/amor.yaml) update network config

### PR DESCRIPTION
**Pull request by dtapiacl**

### CODE REVIEW

File: hieradata/site/cp/role/amor.yaml [hunk 1]
```diff
@@ -5,4 +5,4 @@ docker::networks::networks:
     driver: "macvlan"
     subnet: "139.229.170.0/24"
     gateway: "139.229.170.254"
-    options: ["parent=p2p1"]
+    options: ["parent=ens2f0"]
```
Comment (jhoblitt) [inline hieradata/site/cp/role/amor.yaml:8]: Please add spec coverage for the change. E.g. https://github.com/lsst-it/lsst-control/blob/production/spec/hosts/nodes/azar01.ls.lsst.org_spec.rb#L73-L85

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Most changes should have spec coverage.

File: hieradata/site/cp/role/amor.yaml (hunk 1 above)
Comment (dtapiacl) [reply hieradata/site/cp/role/amor.yaml:8]: Test coverage for the change was incorporated.

File: spec/hosts/nodes/love1.cp.lsst.org_spec.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'love1.cp.lsst.org', :sitepp do
```
Comment (jhoblitt) [inline spec/hosts/nodes/love1.cp.lsst.org_spec.rb]: The hostname is `love01.cp.lsst.org`.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
The hostname should be change to match the real host to avoid confusion in the future. Otherwise, looks good.

**Review by jhoblitt — APPROVED** [review]

**Pull request merged**